### DISCUSSION TITLE
Improve the sql editor

### DISCUSSION THREAD

**Discussion by pitanni** [dbeaver/dbeaver#12774] [Q&A] [answered]
It would be great to have a multi-line selector, as happens for example on notepad++, where by holding down the "Alt" key and selecting the lines with the mouse you can write simultaneously on several lines

**Comment by ShadelessFox** [comment, marked as answer]
Hello @pitanni,

You can activate block selection in <kbd>Edit</kbd> &rArr; <kbd>Toggle Block Selection</kbd> or by pressing <kbd>Alt+Shift+A</kbd>.

Let me know if your request was fulfilled.

**Comment by pitanni** [comment]
Hi @ShadelessFox ,
thank you very much! I did not know this command.
is it possible to select on the last character of each line? (like on DataGrip)

**Comment by ShadelessFox** [comment]
Unfortunately, it is not possible :(
